data/translation/Zed/de_DE.csv

```diff
@@ -0,0 +1,15 @@
+punch-out-catalog.connection.list.title,Ausstanzungskatalog
+punch-out-catalog.connection.add-new-connection,Neu Anschluss
```

Comment:
According to https://documentation.spryker.com/capabilities/back_office/back_office_translations/back-office-translations-overview-201903.htm you should use general sentences as key, not glossary key style.

Reply:
Sorry, I read it several times and do not see a word that we shouldn't use glossary keys.

Reply:
use existing modules as example

Reply:
Existing modules were released before `back-office-translations` feature was released.

I see no benefits to use text as glossary keys, since it will make refactoring harder.
Does it make sense?

Reply:
Discuesed, agreed to get rid of glossary keys